pygments/lexers/scdoc.py

```diff
@@ -0,0 +1,56 @@
+import re
+
+from pygments.lexer import RegexLexer, include, bygroups, \
+    using, this
+from pygments.token import Text, Comment, Keyword, String, \
+    Generic
+
+
+__all__ = ['ScdocLexer']
+
+
+class ScdocLexer(RegexLexer):
```

Comment:
Missing: docstring with description of what scdoc is, and `.. versionadded:: 2.5`